src/components/elements/link.tsx

```diff
@@ -51,7 +51,7 @@ const DrupalLink = ({href, children, ...props}: LinkProps) => {
   }
 
   return (
-    <Link {...props} href={href} className={twMerge("group text-digital-red hocus:text-archway-dark", props.className)}>
+    <Link {...props} href={href} className={clsx("group text-digital-red hocus:text-archway-dark", props.className)}>
```

Comment:
Shouldn't this stay as `twMerge`?

Reply:
When using `twMerge`, `text-*` utility classes are being dropped from the merge with `props.className` 🤔

Reply:
Where would I be able to test that?

Reply:
Hmm, we can see this issue on the live site: https://stanford-university-press-stanford-press.vercel.app/books/law/attacking-judges
(Fix for the typography issue for "Award Winner" and "Also Available from" will be in PR #68 )

`text-18` is being passed in on L144 of the books details page but not appear on the front-end
https://github.com/SU-SWS/press-nextjs/pull/66/files#diff-e9117b32c5de23e51da4444bf52a4bf8be5310e56e93674f973729b53dd448e1L141-L145

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/fe57e31b-c9fe-49bc-ae78-6b258af2ccfd">

Tangentially related to #68 , I think if I could figure out a way to wrangle the twMerge config together so that it can recognize our custom decanter front sizes and prioritize them accordingly, it would allow us to use twMerge and remove the additional checks we have for the headings 🤔

Reply:
We'll put a pin on a good solution and run with this for now. I'll circle back to investigating the twMerge config (hopefully) soon! Perhaps I can tie it into my Professional Development goals as well.